src/moveit_studio_agent_utils/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.16.3)
+project(moveit_studio_agent_utils)
```

Comment:
I would call this package `moveit_studio_agent_examples`, perhaps?

Reply:
Updated, thanks for the suggestion.